postgres/dev/init.sls

```diff
@@ -49,7 +49,7 @@ postgres_maxfiles_limits_conf:
     - group: {{ postgres.group }}
   {% endif %}
 
-  {% if postgres.use_upstream_repo == 'postgresapp' %}
+  {% if 'pkg_repo' in postgres and postgres.use_upstream_repo == 'postgresapp' %}
```

Comment:
This might break MacOS installation, there is a special case of `postgres.use_upstream_repo` value.
The `postgres.pkg_repo` is not defined for it, as I recall. And the `upstream.sls` is not included here.

Reply:
Gotcha. Removed the check.